libraries/rush-lib/src/logic/Autoinstaller.ts

```diff
@@ -133,6 +136,14 @@ export class Autoinstaller {
       // Create file: ../common/autoinstallers/my-task/.rush/temp/last-install.flag
       lastInstallFlag.create();
 
+      FileSystem.writeFile(
+        path.join(nodeModulesFolder, 'rush-autoinstaller.log'),
+        'If this file is deleted, Rush will assume that the node_modules folder has been cleaned and will reinstall it.',
+        {
+          ensureFolderExists: false
+        }
```

Comment:
```suggestion
        'If this file is deleted, Rush will assume that the node_modules folder has been cleaned and will reinstall it.'
```

`ensureFolderExists: false` is the default.